app/views/cases/ico/_case_details.html.slim

```diff
@@ -1,10 +1,10 @@
 .grid-row
-  .column-full.case-details
+  - link_type_prefix = defined?(link_type) && link_type.present? ? "#{link_type}-" : ""
+  div class="column-full #{link_type_prefix}case-details"
     h2.request--heading
-      - if @case.type_abbreviation != case_details.type_abbreviation
-        = "Original case details"
-      - else
-        = "Case details"
+      / i18n-tasks-use t('common.case.case-details')
+      / i18n-tasks-use t('common.case.original-case-details')
```

Comment:
what do these two comment lines do?  Anything special?

Reply:
These signal to i18n-tasks that these translations should be considered used and won't show up in `i18n-tasks unused`. Seems like a better approach than adding an ignore for `common.*case-details` into `i18n-tasks.yml` as it tracks closer to the code where they're used.